cloud_experts_tutorials/cloud-experts-update-component-routes.adoc

```diff
@@ -0,0 +1,252 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-update-component-routes"]
+= Tutorial: Updating component routes with custom domains and TLS certificates
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-update-component-routes
+
+toc::[]
+
+:fn-supported-versions: footnote:[Modifying these routes on {product-title} ROSA versions prior to 4.14 is not typically supported. However, if you have a cluster using version 4.13, you can request for Red Hat Support to enable support for this feature on your version 4.13 cluster by link:https://access.redhat.com/support/cases/new[opening a support case].]
+:fn-term-component-routes: footnote:[We use the term "component routes" to refer to the `OAuth`, `Console`, and `Downloads` routes that are provided when ROSA are first installed.]
+
+//Article text
+This guide demonstrates how to modify the hostname and TLS certificate of the Web console, OAuth server, and Downloads component routes in {product-title} (ROSA) version 4.14 and above.{fn-supported-versions}
+
+The changes that we make to the component routes{fn-term-component-routes} in this guide are described in greater detail in the customizing the link:https://docs.openshift.com/container-platform/latest/authentication/configuring-internal-oauth.html#customizing-the-oauth-server-url_configuring-internal-oauth[internal OAuth server URL], link:https://docs.openshift.com/container-platform/latest/web_console/customizing-the-web-console.html#customizing-the-console-route_customizing-web-console[console route], and link:https://docs.openshift.com/container-platform/latest/web_console/customizing-the-web-console.html#customizing-the-download-route_customizing-web-console[download route] OpenShift Container Platform documentation. 
+
+[id="prerequisites_{context}"]
+== Prerequisites
+* ROSA CLI (`rosa`) version 1.2.37 or higher
+* AWS CLI (`aws`)
+* A ROSA cluster
+* OpenShift CLI (`oc`)
+* `jq` CLI
+* Access to the cluster as a user with the `cluster-admin` role.
+* OpenSSL (for generating the demonstration SSL/TLS certificates)
+
+[id="environment-setup_{context}"]
+== Setting up your environment
+
+. Configure an environment variable for your cluster name:
++
+[NOTE]
+====
+You must be logged in as an administrator.
```

Comment:
Rather than putting this in a note, should we just add a first step?

```
. Log in to your cluster using an account with cluster-admin privileges.
+
[source,terminal]
----
$ rosa login --token=<cluster-admin_token>
----
```